data/human/free worlds 2 middle.txt

```diff
@@ -659,7 +659,7 @@ mission "FW Southern Prisoners - Released"
 			`<origin> has become the home of a major Navy base almost overnight; the number of ships and people here is almost overwhelming. You return the prisoners, who have all taken an oath of parole to take no further part in the war against the Free Worlds. One of the four Navy admirals who you and Alondo met with earlier is here, and as you are unloading the prisoners, he says, "More volunteers for the 'Oathkeepers,' I see."`
 			choice
 				`	"Who are the 'Oathkeepers'?"`
-			`	"The officers you granted parole to months ago. We reassigned them all to duty in the north, patrolling for pirates."`
+			`	"The officers you granted parole two months ago. We reassigned them all to duty in the north, patrolling for pirates."`
```

Comment:
I think this was fine before.
```suggestion
			`	"The officers you granted parole to months ago. We reassigned them all to duty in the north, patrolling for pirates."`
```

Reply:
I think it was quite a lot more than two months prior to this point.

Reply:
Ah, thank you, that one got me, non-native speaker... maybe the sentence can be rearranged to make it clearer? It's not a complete sentence anyway (which is fine... it's quoting verbal communication after all).

Reply:
I think both "granted parole two months ago" and "granted parole to months ago" are both grammatically correct, but "two" is logically incorrect because that event was, I think, at least 8 months ago. It's also possible to take considerably longer than that. I don't think it'd be particularly difficult to get to this point over a year ago, so perhaps rewording it more to just "granted parole previously" would be the best option?

Reply:
My comment wasn't on correctness (anymore). You might have read it wrong but it would actually be a good change from my PoV. I am not sure if it would be grammatically OK but even removing the "to" would make it easier to understand (for me anyway).